### PR TITLE
1.0 4 release

### PR DESCRIPTION
**Pull request by ferasd**
@haggaie 
@drossetti

### CODE REVIEW

File: build_module.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+echo "Building source rpm for nvidia_peer_memory..."
+mkdir -p $tmpdir/topdir/{SRPMS,RPMS,SPECS,BUILD}
+ex "rpmbuild -ts --nodeps --define '_topdir $tmpdir/topdir' --define 'dist %{nil}' --define '_source_filedigest_algorithm md5' --define '_binary_filedigest_algorithm md5' $tmpdir/nvidia_peer_memory-$VERSION.tar.gz >/dev/null"
+srpm=`ls -1 $tmpdir/topdir/SRPMS/`
+mv $tmpdir/topdir/SRPMS/$srpm /tmp
+
+echo "Building debian tarball for nvidia-peer-memory..."
+ex mv $tmpdir/nvidia_peer_memory-$VERSION $tmpdir/nvidia-peer-memory-$VERSION
+# update version in changelog
+sed -i -r "0,/^(.*) \(([a-zA-Z0-9.-]+)\) (.*)/s//\1 \(${VERSION}-${RELEASE}\) \3/" $tmpdir/nvidia-peer-memory-${VERSION}/debian/changelog
```
Comment (drossetti) [inline build_module.sh]: why do you need to update the debian version in the script ?

File: build_module.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+ex cp -r . $tmpdir/nvidia_peer_memory-$VERSION
+pushd $tmpdir > /dev/null
+ex tar czf nvidia_peer_memory-$VERSION.tar.gz nvidia_peer_memory-$VERSION --exclude=.* --exclude=build_release.sh
+popd > /dev/null
+
+echo
+echo "Building source rpm for nvidia_peer_memory..."
+mkdir -p $tmpdir/topdir/{SRPMS,RPMS,SPECS,BUILD}
+ex "rpmbuild -ts --nodeps --define '_topdir $tmpdir/topdir' --define 'dist %{nil}' --define '_source_filedigest_algorithm md5' --define '_binary_filedigest_algorithm md5' $tmpdir/nvidia_peer_memory-$VERSION.tar.gz >/dev/null"
```
Comment (drossetti) [inline build_module.sh]: this script might be invoked on a Debian based OS. 
Can't you simply print out this as the recipe for RPM distros ?

Comment (ferasd) [inline build_module.sh]: Changed the script to build sources only for the local distribution

File: build_module.sh (hunk 1 above)
Comment (ferasd) [inline build_module.sh]: This part is not needed, removed.

**Pull request merged**